### PR TITLE
Man formatting fixes

### PR DESCRIPTION
**Pull request by slusarz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,11 +14,14 @@
     "pagefind": "^1.1.0",
     "path-to-regexp": "^6.2.2",
     "pdc": "^0.2.3",
-    "remark-gfm": "^4.0.0",
+    "remark-definition-list": "^2.0.0",
     "remark-man": "^9.0.0",
     "remark-parse": "^11.0.0",
     "unified": "^11.0.5",
+    "unist-builder": "^4.0.0",
```
Comment (sirainen) [inline package.json:21]: package* unrelated changes

**Review by sirainen — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (slusarz) [reply package.json:21]: These are needed for util/generate_man.js.  4 package dependencies were added and 1 was removed.

**Comment by slusarz** [comment]
Also, confirmed that the multi-level indention output issue is a problem with remark-man code, not the added code.  (It loses the indent size for an indent at the end of a sub-paragraph.)

File: package.json (hunk 1 above)
Comment (sirainen) [reply package.json:21]: okay, but what about package-lock.json with a billion changes?

Comment (slusarz) [reply package.json:21]: Those new packages apparently have a billion sub-dependencies.

**Review by sirainen — APPROVED** [review]
